src/api/app/views/shared/_package_branch_form.html.erb

```diff
@@ -0,0 +1,17 @@
+<p>
+  <% if show_source_fields == true %>
```

Comment:
I already changed this. It had some background why I did this like it is in that commit.